kaldb/src/main/java/com/slack/kaldb/server/Kaldb.java

```diff
@@ -61,7 +61,7 @@ public static void main(String[] args) throws IOException {
     kalDb.setup();
   }
 
-  public void setup() {
+  public void setup() throws Exception {
```

Comment:
This no longer throws a checked exception.

Reply:
Fixed.